packages/anima-cli/src/commands/sync.ts

```diff
@@ -28,224 +26,131 @@ export const builder: CommandBuilder = (yargs) =>
   yargs
     .options({
       token: { type: 'string', alias: 't' },
-      directory: { type: 'string', alias: 'd' },
+      storybook: { type: 'string', alias: 's' },
```

Comment:
regarding the `--storybook` argument being like a boolean but also a string, would this work? 
```suggestion
      storybook: {
        alias: 's',
        default: 'storybook-static',
        type: 'string'
      },
```
basically it has a default, so if you don't pass anything it assumes the default? 
@Datlyfe @Kubessandra

Reply:
Yes, but if you pass `--storybook` it will wait an argument

Reply:
what about using coerce? 😬 
```suggestion
      storybook: {
              alias: 's',
              default: 'storybook-static',
              coerce: (arg) => {
                    if (arg === true) {
                      return 'storybook-static'; // Set a default value if it's a boolean
                    } else {
                      return arg; // Return the string argument value
                    }
               }
      },
```

Reply:
Did you try ?
I will check that, did not had the time yet 😌

Reply:
I only did a quick test before posting:
<img width="1167" alt="image" src="https://github.com/AnimaApp/anima-storybook-cli/assets/1323193/37a153c5-3af3-4859-8ba8-4e0d59b26c56">
https://stackblitz.com/edit/node-xau9pm?file=index.js